### PR TITLE
w/series-3/cover/dev: merge with prod

### PR DESCRIPTION
**Pull request by nmelnychenko**

### CODE REVIEW

File: src/component.js [hunk 1]
```diff
@@ -55,8 +54,8 @@ class Block extends React.Component {
                 tagName="h2"
                 className={classNames(
                   style.heading,
-                  this.ui('ui-tex-center'),
-                  this.ui('ui-heading-mp'),
+                  this.props.$theme.enabled('ui-tex-center'),
```
Comment (t-nick) [inline src/component.js]: ('ui-tex-center') заменить на ('ui-text-center')

File: src/component.js [hunk 2]
```diff
@@ -66,8 +65,8 @@ class Block extends React.Component {
                 tagName="p"
                 className={classNames(
                   style.subheading,
-                  this.ui('ui-tex-center'),
-                  this.ui('ui-subheading-mp'),
+                  this.props.$theme.enabled('ui-tex-center'),
```
Comment (t-nick) [inline src/component.js]: ('ui-tex-center') заменить на ('ui-text-center')

Comment (nmelnychenko) [reply src/component.js]: fixed

File: src/component.js (hunk 1 above)
Comment (nmelnychenko) [reply src/component.js]: fixed

**Pull request merged**